launcher-scripts/Linux-launcher.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+OLD_JAVA_HOME=$JAVA_HOME
```

Comment:
Have you tested this? I'm concerned that without the curly braces you don't get the value of the calling environment.